NitroxClient/MonoBehaviours/Gui/HUD/RemotePlayerVitals.cs

```diff
@@ -76,7 +76,11 @@ public void LateUpdate()
             waterBar?.UpdateVisual();
 
             // Make canvas face camera.
-            canvas.transform.forward = Camera.main.transform.forward;
+            Camera camera = Camera.main;
```

Comment:
Maybe move this outside the loop? Camera.main is a little resource heavy as far as I know...

Reply:
True, but the camera changes so can't really do this effectively. Already tried to do this.